.ci/containers/membership-checker/reviewer_assignment.go

```diff
@@ -144,25 +129,30 @@ func requestPullRequestReviewer(prNumber, assignee, GITHUB_TOKEN string) error {
 	return nil
 }
 
-func requestRandomReviewer(prNumber, GITHUB_TOKEN string) error {
-	assignee := getRandomReviewer()
-	err := requestPullRequestReviewer(prNumber, assignee, GITHUB_TOKEN)
-	if err != nil {
-		return err
+func formatReviewerComment(newPrimaryReviewer string, authorUserType userType, trusted bool) string {
```

Comment:
Hm, I think you're right that this could cause problems - for example, if we wanted to compile this as a binary in the future. We can stick to embedding and then parse the embedded string instead.